src/puppetlabs/jdbc_util/pglogical.clj

```diff
@@ -18,13 +19,16 @@
   pglogical.replicate_ddl_command, escaping quotes and wrapping the statement so
   it won't return anything."
   [sql schema]
-  (str "do 'begin perform "
-       (unsafe-escape-sql-quotes
-        (str "pglogical.replicate_ddl_command('"
-             "set local search_path to " schema "; "
-             (unsafe-escape-sql-quotes sql)
-             "');"))
-       " end;';"))
+  (let [effective-sql (if (sequential? sql)
+                        (str (str/join "; " (map identity sql)) ";")
```

Comment:
what is map identity doing? That should be a no-op

Reply:
Yes, that was a mistake.  Will remove.